website_chd_product_configurator/templates/templates.xml

```diff
@@ -0,0 +1,369 @@
+                                 </thead>
+                                 <tbody>
+                                    <tr>
+                                        <td>
+                                         Width:
+                                        </td>
+                                        <td>
+                                        From <t t-esc="curr_product_id.chd_min_width"/> to <t t-esc="curr_product_id.chd_max_width"/>
+                                        </td>
+                                    </tr>
+                                      <tr>
+                                        <td>
+                                          height:
+                                        </td>
+                                        <td>
+                                         From <t t-esc="curr_product_id.chd_min_height"/> to <t t-esc="curr_product_id.chd_max_height"/>
+                                        </td>
+                                    </tr>
+                                 </tbody>
+                                 </table>
+                            </div>
+
+                        </div>
+                    </div>
+                    <div class="unselected-field" style="display: inline-block;">
+                         <label>Type:
+                         <select t-attf-name="type" class="form-control type_selection" id="type_select_id">
+                                <option></option>
+                                <t t-foreach="curr_types" t-as="curr_type">
+                                    <option t-att-value="curr_type.id" class="form-control type_selection">
+                                         <t t-esc="curr_type.name"/>
+                                    </option>
+                                </t>
+                          </select>
+                        </label>
+                    </div>
+                    <div class="unselected-field" style="display: inline-block;">
+                         <label>Finishing:
+                         <select t-attf-name="finishing" class= "form-control fini_selection" id="fini_select_id">
+                            <!-- filled by js -->
+                          </select>
+                        </label>
+                    </div>
+
+
+
+
+                    <!-- generation of product price attributes, all required -->
+                    <div class="form group">
+                              <div class="col-xs-12">
+                                   <h3 t-if="len(curr_chd_price_component_ats) &gt; 1"> Additional parameters for product (required) </h3>
+                                        <t t-foreach="curr_chd_price_component_ats" t-as="curr_chd_price_component_at">
+                                            <!-- i am exluding quantity -->
+                                            <div t-if="curr_chd_price_component_at.property_id.name != 'Quantity'" class="row">
```

Comment:
Does this property exist as xmlid in the database? If yes, use that (otherwise, you might run into problems with translations, renamed property, etc)